4-reduction/scipp-introduction.ipynb

```diff
@@ -2255,25 +2255,6 @@
     "fig = pp.plot({'Manhattan': mean_manh, 'JFK': mean_jfk})\n",
     "fig"
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "8dfeaa3f-4090-46b9-b212-84c788779dc3",
-   "metadata": {
-    "editable": true,
-    "slideshow": {
-     "slide_type": ""
-    },
-    "tags": [
-     "remove-input",
-     "hide-output"
-    ]
-   },
-   "outputs": [],
-   "source": [
-    "fig.fig"
```

Comment:
I believe this was intentional, with a combination of tags, to hide the input, show the output?
But I can't remember what we were trying to do.

I think it should just be that the whole cell (including output) is hidden and can be toggled in the rendered book (as in the "Rush hours" exercise above).

So you in addition to removing the cell here, we need to remove the `"remove-output"` tag in the cell above (I think?).

Finally, I think we might need to do the same for the previous exercise "Expensive hours"

Reply:
Is there a way to remove it from the 'workbook' instead?

Reply:
Setting the tags as
```
        "hide-cell",
        "solution",
```
basically does this? It will show as a toggle section in the built book, and be removed from the workbook.

Reply:
I added the `dmsc-school-remove` tag which to my understanding removes the cell from workbooks but has no other impact.